WarOfEmpires.Models/Empires/TroopModel.cs

```diff
@@ -22,8 +22,8 @@ public sealed class TroopModel {
         [RegularExpression("^\\d{0,6}$", ErrorMessage = "Footman mercenaries must be a valid number")]
         public string MercenaryFootmen { get; set; }
         [DisplayName("Stamina")]
-        [RegularExpression("^\\d{0,6}$", ErrorMessage = "Stamina must be a valid number")]
-        public string Stamina { get; set; }
+        [RegularExpression("^\\d{0,6}$", ErrorMessage = "Stamina to heal must be a valid number")]
```

Comment:
As discussed, if you want to make stamina to heal required in the client-side validation, the regex needs to change. This one allows min. 0 numbers but it should be min. 1.

Reply:
Regular Expression updated to "^\\d{1,3}$" (Since stamina can never be more than 100, allowing more than 3 digits is not necessary)